### DISCUSSION TITLE
Customizable Status Bar

### DISCUSSION THREAD

**Discussion by mahume** [sxyazi/yazi#686] [1. Q&A]
I'm checking out the default theme at `yazi-config/preset/theme.toml`. Is there a way to customize the status bar beyond the font attributes? An ability to move elements to different positions, selective choose which elements to show, maybe change the pill look into a rectangle. Or is this delegated to the plugin system?

**Comment by sxyazi** [comment]
The theme system currently only supports customization for basic styles. As for "change the pill look into a rectangle", I think you can achieve this by using `separator_open` and `separator_close`, see https://yazi-rs.github.io/docs/configuration/theme#status

For further customization, you can achieve it through the UI plugin, here are a few examples:

- [No status bar](https://yazi-rs.github.io/docs/tips#no-status-bar)
- [Show symlink in status bar](https://yazi-rs.github.io/docs/tips#show-symlink-in-status-bar)
- [Show user/group of files in status bar](https://yazi-rs.github.io/docs/tips#show-symlink-in-status-bar)

See https://github.com/sxyazi/yazi/blob/main/yazi-plugin/preset/components/status.lua for the full status component code.
